media_driver/agnostic/common/vp/hal/vphal_render_vebox_memdecomp.cpp

```diff
@@ -663,7 +663,37 @@ MOS_STATUS MediaVeboxDecompState::GetResourceInfo(PMOS_SURFACE surface)
         &resDetails));
 
     surface->Format                                             = resDetails.Format;
+#ifdef LINUX
```

Comment:
@XinfengZhang Can we add such macro?

Reply:
yes, we can. We can add such macro in mos layer. you can find such macro on this cpp file.